client/buildpacks.go

```diff
@@ -114,6 +118,16 @@ func (b *buildpacksFunctionServer) build(ctx context.Context) error {
 	return nil
 }
 
+var runtimeLanguageRegexp = regexp.MustCompile(`^[a-zA-Z]+`)
+
+func (b *buildpacksFunctionServer) buildpackBuilderImage() (string, error) {
+	runtimeLanguage := runtimeLanguageRegexp.FindString(b.runtime)
+	if runtimeLanguage == "" {
+		return "", fmt.Errorf("Invalid runtime format. Runtime should start with language followed by version. Example: go119, python311. Got %q", b.runtime)
```

Comment:
I _believe_ gcr.io/buildpacks/builder is the GCP builder that contains all language builders, the one you're changing to is the unified language builder, which is what we're using in prod. The URL should be in the form of `gcr.io/gae-runtimes/buildpacks/<language>/builder` and <language> should be just "go", "java" and so on.

Reply:
Hey Kayla, thanks for taking a look. I am extracting the runtime language name (go, php) from the runtime (go119, php82) input stored in b.runtime and using that to fill the `<language>` portion of the url.

Reply:
does the builder not automatically detect the language? Do you know if these other builders work for php ruby integ tests? https://cloud.google.com/docs/buildpacks/builders

Reply:
So, we have this generic builder at "gcr.io/gae-runtimes/buildpack/builder" that can build all languages that we call the "OSS builder". It's not maintained as well as the GCF/GAE builders, so we never actually added support for PHP functions to that builder, which is why PHP and Ruby aren't working.

You'll notice that there's no `google.php.functions-framework` buildpack mentioend in this builder's configuration:
https://github.com/GoogleCloudPlatform/buildpacks/blob/main/builders/gcp/base/builder.toml

The builders we actually use for GCF/GAE are language-specific and use the <language>/builder URL pattern that Kenneth is updating in this PR: https://github.com/GoogleCloudPlatform/buildpacks/blob/main/builders/php/builder.toml

Using those are better for the FF tests because it's closer to what is actually used in production. When I originally added the buildpack integration tests to FF, the URL for the language-specific builders were in flux, so I just used the "OSS builder" to reduce complexity.